<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beancount importer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QPushButton" name="selectExpensesAccountBtn">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Transactions</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="selectPaymentAccountBtn">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Default currency</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Default payment account</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="defaultPaymentAccountLE"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Default expenses account</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QTableView" name="transactionTableView"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="defaultExpensesAccountLE"/>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="defaultCurrencyComboBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Import to</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QLineEdit" name="importToPathLE"/>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="importBtn">
      <property name="text">
       <string>Import</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openBeancountAccountAction"/>
    <addaction name="openAlipayCsvAction"/>
    <addaction name="openWechatCsvAction"/>
   </widget>
   <widget class="QMenu" name="menuAccount_Map">
    <property name="title">
     <string>Account Map</string>
    </property>
    <addaction name="payeeToAccountAction"/>
    <addaction name="billAccountToFromAccountAction"/>
    <addaction name="separator"/>
    <addaction name="trainPayeeToAccountMapAction"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuAccount_Map"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openAlipayCsvAction">
   <property name="text">
    <string>Open Alipay csv</string>
   </property>
  </action>
  <action name="openBeancountAccountAction">
   <property name="text">
    <string>Open beancount</string>
   </property>
  </action>
  <action name="openWechatCsvAction">
   <property name="text">
    <string>Open Wechat csv</string>
   </property>
  </action>
  <action name="payeeToAccountAction">
   <property name="text">
    <string>Payee to account</string>
   </property>
  </action>
  <action name="billAccountToFromAccountAction">
   <property name="text">
    <string>Bill account to From account</string>
   </property>
  </action>
  <action name="trainPayeeToAccountMapAction">
   <property name="text">
    <string>Train Payee to account map</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
